### PR TITLE
Updating Zscaler playbooks to add in Solution

### PR DESCRIPTION
**Pull request by v-sabiraj**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-vdixit** [comment]
Hi @v-sabiraj please fix the failing validations, thanks!

**Comment by v-sabiraj** [comment]
> Hi @v-sabiraj please fix the failing validations, thanks!

fixed

**Comment by v-vdixit** [comment]
Hi @v-sabiraj, please fix the failing validations, thanks!

**Comment by v-atulyadav** [comment]
Hi @v-sabiraj, please fix the failing validations, thanks!

**Comment by v-vdixit** [comment]
Hi @v-sabiraj please provide update on this, thanks!

**Review by anki-narravula — CHANGES_REQUESTED** [review]

File: Solutions/Zscaler Internet Access/Playbooks/Authentication/azuredeploy.json [hunk 1]
```diff
@@ -2,14 +2,19 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```
[the remark this answers is not in the document]
Comment (v-sabiraj) [reply Solutions/Zscaler Internet Access/Playbooks/Authentication/azuredeploy.json]: Fixed.

File: Solutions/Zscaler Internet Access/Playbooks/Authentication/azuredeploy.json [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
     "contentVersion": "1.0.0.0",
     "metadata": {
```
[the remark this answers is not in the document]
Comment (v-sabiraj) [reply Solutions/Zscaler Internet Access/Playbooks/Authentication/azuredeploy.json]: image
This is mentioned in prerequisites.

image

Also it is mentioned in dependencies.

File: Solutions/Zscaler Internet Access/Playbooks/Add-Url-To-Category/azuredeploy.json [hunk 3]
```diff
@@ -2,15 +2,17 @@
...
-        "title": "Block URL - Zscaler", 
+        "title": "Block URL - Zscaler",
         "description": "This playbook allows blocks URLs in Zscaler by adding them to categories",
-        "prerequisites": ["1. This playbook leverages a nested playbook: Zscaler authentication. Deploy this playbook before deploying this template in the same resource group.",
-        "2. Create a block category in Zscaler, which this playbook will add IPs to."],
-        "lastUpdateTime": "2021-07-28T00:00:00.000Z", 
-        "entities": ["Url"], 
-        "tags": ["Remediation"], 
+        "prerequisites": [
+            "1. This playbook leverages a nested playbook: Zscaler authentication. Deploy this playbook before deploying this template in the same resource group. [Click here](https://portal.azure.com/#create/Microsoft.Template/uri/https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FPlaybooks%2FZscaler%2FAuthentication%2Fazuredeploy.json) to deploy Zscaler authentication playbook",
```
[the remark this answers is not in the document]
Comment (anki-narravula) [reply Solutions/Zscaler Internet Access/Playbooks/Add-Url-To-Category/azuredeploy.json:8]: yes

File: Solutions/Zscaler Internet Access/Playbooks/Authentication/azuredeploy.json [hunk 4]
```diff
@@ -2,14 +2,19 @@
...
+        "entities": [],
+        "tags": [
+            "Utilities"
+        ],
         "support": {
-            "tier": "community" 
+            "tier": "community"
         },
         "author": {
             "name": "Accenture"
```
[the remark this answers is not in the document]
Comment (v-sabiraj) [reply Solutions/Zscaler Internet Access/Playbooks/Authentication/azuredeploy.json]: @anki-narravula, fixed.

**Review by anki-narravula — APPROVED** [review]
Looks good. Approved.

**Pull request merged**